### PR TITLE
some development of MSCWCtm reader

### PR DESCRIPTION
**Pull request by hansbrenna**
A fix extending to adding some features to the EMEP reader

### CODE REVIEW

**Comment by jgliss** [comment]
@hansbrenna can you change the base of this PR to branch aeroval-updates (or main-dev)? I would like to get rid of av-trend-patch, to prepare release of 0.12.0. Cheers

**Comment by jgliss** [comment]
Or, if it is ready, we can of course merge this before merging #400

**Comment by hansbrenna** [comment]
I need to check the failing tests. Then it should be ready

**Comment by jgliss** [comment]
@hansbrenna could you add tests for the new features? So that coverage does not decrease?

**Comment by hansbrenna** [comment]
OK if I merge this now @jgliss ?

File: pyaerocom/colocation_auto.py [hunk 1]
```diff
@@ -1027,6 +1027,8 @@ def _get_ts_type_read(self, var_name, is_model):
         tst = self.ts_type # default
         if is_model and self.model_ts_type_read is not None:
             tst = self.model_ts_type_read
+            if tst == '':
+                tst = self.ts_type
```
Comment (jgliss) [inline pyaerocom/colocation_auto.py:1031]: Some of the colocation tests seem to fail, is this maybe because of this?

**Review by jgliss — COMMENTED** [review]
Looks good in general, but please check and fix the failing CI before merging.

**Comment by hansbrenna** [comment]
This is a bit weird. None of the tests in test_colocation_auto.py fail when I run them locally...

**Comment by jgliss** [comment]
> This is a bit weird. None of the tests in test_colocation_auto.py fail when I run them locally...

hmm, seems to be only for certain python versions...

**Comment by hansbrenna** [comment]
Have you seen that before?

**Comment by jgliss** [comment]
No, not really, could be worthwhile to checkout the individual runs and what exactly went wrong there (if you click on "Details" you can see the console output of the CI run.

**Comment by hansbrenna** [comment]
hmm. It fails on 3.8 now which is the same version I'm using locally. I tested these test locally and they all ran fine.

FAILED pyaerocom/test/test_colocation_auto.py::test_colocator__infer_start_stop_yr_from_model_reader
FAILED pyaerocom/test/test_colocation_auto.py::test_colocator_with_obs_data_dir_ungridded
FAILED pyaerocom/test/test_colocation_auto.py::test_colocator_with_model_data_dir_ungridded
FAILED pyaerocom/test/test_colocation_auto.py::test_colocator_with_obs_data_dir_gridded
====== 4 failed, 905 passed, 42 skipped, 68 warnings in 79.67s (0:01:19) =======

**Review by jgliss — APPROVED** [review]
Magic :) good to go!

**Pull request merged**